documentation/docs/installation.md

```diff
@@ -21,6 +21,11 @@ You will need an `.env` file to run it. We will configure it in the next step.
 To access Quix, navigate to:  
 `http://localhost:3000`
 
+## Clone Repo
+Clone Git Repository
```

Comment:
can remove this I think

Reply:
leave without any title? or just subtitle

Reply:
Leave just `## Clone repository`